<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSTK Lab Manager</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="trialParamsLayout">
           <item>
            <widget class="QLabel" name="trialParamsLabel">
             <property name="text">
              <string>Run Parameters:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="trialParamsTableView">
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="autoScroll">
              <bool>true</bool>
             </property>
             <property name="verticalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>57</number>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>21</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="sensorTreeLayout">
           <item>
            <widget class="QLabel" name="sensorTreeLabel">
             <property name="text">
              <string>Available Sensors:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeView" name="sensorTreeView">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="autoScrollMargin">
              <number>20</number>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="verticalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="indentation">
              <number>10</number>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="headerMinimumSectionSize">
              <number>64</number>
             </attribute>
             <attribute name="headerDefaultSectionSize">
              <number>128</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="PlotWidget" name="sensorPlotWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
        <item>
         <widget class="QPushButton" name="startDiscoveryButton">
          <property name="text">
           <string>Discover (3s)</string>
          </property>
          <property name="icon">
           <iconset theme="bluetooth">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="recordButton">
          <property name="text">
           <string>Record Data</string>
          </property>
          <property name="icon">
           <iconset theme="media-record">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearDataButton">
          <property name="text">
           <string>Clear Data</string>
          </property>
          <property name="icon">
           <iconset theme="edit-delete">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportDataButton">
          <property name="text">
           <string>Export Data</string>
          </property>
          <property name="icon">
           <iconset theme="go-up">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="importDataButton">
          <property name="text">
           <string>Import Data</string>
          </property>
          <property name="icon">
           <iconset theme="go-down">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>&amp;Configure</string>
    </property>
    <addaction name="actionLoad_Config"/>
    <addaction name="actionView_Log"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
   </widget>
   <addaction name="menuConfigure"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <action name="actionLoad_Config">
   <property name="text">
    <string>&amp;Load Config...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionView_Log">
   <property name="text">
    <string>&amp;View Log...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
